src/core/componentSystems/AnimationSystem.js

```diff
@@ -101,24 +103,95 @@ export class AnimationSystem extends MRSystem {
 
     /**
      * @function
-     * @description Sets the Animation of the entity object based on the component value associated with it.
+     * @description Sets the Animation of the entity object based on the component value associated with it. Otherwise lets those
+     * be handled by the threejs default setup. (Always looping, always playing based on browser type, etc).
      * @param {object} entity - the entity being updated based on the component being detached.
-     * @param {object} comp - component that contains a string value of 'play', 'pause', 'stop'
+     * @param {object} comp - component that contains the values of 'action', 'loop', and/or 'loopMode'
      */
     setAnimation(entity, comp) {
         let clip = entity.animations[comp.clip];
-        switch (comp.action) {
-            case 'play':
-                entity.mixer.clipAction(clip).play();
-                break;
-            case 'pause':
-                entity.mixer.clipAction(clip).pause();
-                break;
-            case 'stop':
-                entity.mixer.clipAction(clip).stop();
-                break;
-            default:
-                break;
+        let action = entity.mixer.clipAction(clip);
+
+        if (comp.hasOwnProperty('action')) {
```

Comment:
You can also do `comp?.action` which is a little cleaner.

Reply:
i like the simpler approach you note, but that also makes it turn into a truthy check of the `action` value if it exists, meaning it can silently error (instead of hitting the default case) if a user makes a typo and leaves it as a 0 or an empty string for example

i'd prefer to keep it as the hasOwnProperty line, but lmk your thoughts

Reply:
hmm does comp?.action == null not hit the default state?

Reply:
sorry im confused - all that does is check if comp exists - and then whether comp.action == null

in the `comp.hasOwnProperty` line im trying to check if action exists before running through our switch statements - do you have a different preferred way of writing that?

in the audio system it looks like that's setup as `comp.action ?? false` - is that what you'd prefer instead?

Reply:
i think i'd still be a little worried of the `comp.action` value existing but being incorrect as a truthy-ness option and not going into if-statement 🤔 

thoughts?

Reply:
i'm going to merge it in since approved so not blocker, but lmk since that's a quick pr fix afterwards if needed